doc/conf.py

```diff
@@ -1,4 +1,17 @@
-# -*- coding: utf-8 -*-
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
is there a reason we have 2017-2018 for some and just 2018 for others?

Reply:
Not really, I will update all the copyright to the 2017-2018 range.